config/initializers/omniauth.rb

```diff
@@ -15,9 +17,11 @@
   dfe_sign_in_issuer_uri = URI(ENV.fetch("DFE_SIGN_IN_ISSUER", "example"))
 
   Rails.application.config.middleware.use OmniAuth::Builder do
-    provider :openid_connect,
+    provider :dfe_openid_connect,
              name: :dfe,
              callback_path: "/check-records/auth/dfe/callback",
+             logout_path: "/sign-out",
+             post_logout_redirect_uri: "#{ENV['CHECK_RECORDS_DOMAIN']}/check-records",
```

Comment:
This isn't currently a valid route in the app — we could change it to one of these perhaps:

```suggestion
             post_logout_redirect_uri: "#{ENV['CHECK_RECORDS_DOMAIN']}/check-records/search",
```


```suggestion
             post_logout_redirect_uri: "#{ENV['CHECK_RECORDS_DOMAIN']}/check-records/sign-in",
```

Alternatively, we could add this to the Check routes (within the namespace block):
  `get "/", to: redirect("/check-records/search")`

Reply:
Ah OK. I was going by what is configured in the Sign In Manage configuration (all logout redirect URLs end in `/check-records`). I had assumed these were valid paths. I'll update Manage to point to `/check-records/search`.

Reply:
I think this is an issue which also affects the termination of the application session.
So the redirect needs to be the sign out path on the application.
This would mean that the loop is:
Click _Sign out_
-> sign out of DfE Sign in
-> redirect to application sign out path (end of application session)
-> redirect to sign in path on app

Reply:
I've amended the `post_logout_redirect_uri` value in `config/initializers/omniauth.rb` and updated the corresponding URLs in the dev Manage Sign in service.

Reply:
@steventux looks good, seeing a sign-in prompt from DSI after signing out of Check and then signing in again. 👍🏼